content/docs/command-reference/commit.md

```diff
@@ -46,6 +40,20 @@ scenarios are further detailed below.
   reproduce the whole pipeline. If you're sure no pipeline results would change,
   use `dvc commit` to force update the `dvc.lock` or `.dvc` files and cache.
 
+- In cases where we have previously executed a stage (either by writing `dvc.yaml`
+  manually and using `dvc repro`, or with `dvc run`), and later notice that
+  output files or directories created by the stage command, which are already in
+  the workspace, are missing from `dvc.yaml` (`outs` field). We can
```

Comment:
Why not just use the exact same text we have previously reviewed and agreed upon? As requested in https://github.com/iterative/dvc.org/pull/1840#pullrequestreview-512176615:

`that some of the output files or directories it creates, which are already in the <abbr>workspace</abbr>`